docs/sample_config.yaml

```diff
@@ -1351,3 +1351,13 @@ password_config:
 #    alias: "*"
 #    room_id: "*"
 #    action: allow
+
+
+# Server admins can define a Python module that implements extra rules for
+# allowing or denying incoming events. In order to work, this module needs to
+# override the methods defined in synapse/events/third_party_rules.py.
+#
```

Comment:
Can you add a note here that this only works for rooms in closed federations, where each server has the same rules? Otherwise people might try and use it for other purposes.